src/store/preferences.ts

```diff
@@ -206,6 +206,22 @@ export const actions = {
 
     return severities;
   },
+  async toggleShowMuted({ dispatch, rootState }: PrefActionContext, isMuted: boolean) {
```

Comment:
This is a `set` rather than a `toggle` (because it doesn't depend on the current state), right?